### PR TITLE
WIP docs for ubernetes-lite

### PR DESCRIPTION
**Pull request by justinsb**
WIP for discussion.  Mostly so we have a set of docs for actually using ubernetes-lite, which we can build up to user-friendly documentation.

### CODE REVIEW

**Comment by davidopp** [comment]
Also please be sure to explain whether the master components are running in a replicated/HA configuration, since I assume one of the primary reasons people might use Ubernetes Lite is to be able to survive zone failure.

File: docs/getting-started-guides/ubernetes-lite.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    fsType: "ext4"
+EOF
+```
+
+The PV is also tagged with the zone & region is is in (us-central1-a):
+
+```
+> kubectl get pv
+NAME           LABELS                                                                                                        CAPACITY   ACCESSMODES   STATUS      CLAIM     REASON    AGE
+my-data-disk   failure-domain.alpha.kubernetes.io/region=us-central1,failure-domain.alpha.kubernetes.io/zone=us-central1-a   5Gi        RWO           Available                       5s
```
Comment (quinton-hoole) [inline docs/getting-started-guides/ubernetes-lite.md]: This has been changed from alpha to beta.

https://github.com/kubernetes/kubernetes/blob/master/pkg/api/unversioned/well_known_labels.go#L20

File: docs/getting-started-guides/ubernetes-lite.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+```
+KUBE_USE_EXISTING_MASTER=true MULTIZONE=1 KUBE_GCE_ZONE=us-central1-b cluster/kube-up.sh
+```
+
+View the nodes again; 3 more nodes should have launched and be tagged in us-central1-b:
+
+```
+kubectl get nodes
+```
```
Comment (quinton-hoole) [inline docs/getting-started-guides/ubernetes-lite.md]: Suggest adding the following here to address @davidopp 's comment about highly available master components.  Then we should merge this, and flesh it out further in a separate PR (that I can work on if you're still busy with other things @justinsb ):

```
Note that the default kube-up.sh script currently still creates all master components in a single availability zone, so if that particular zone fails, the master components will experience an outage until that zone is restored.  If you require a highly available set of master components, please follow the documented steps to achieving [high ava](https://github.com/kubernetes/kubernetes/blob/master/docs/admin/high-availability.md) But the pods in other zones will continue to run, and service load-balancers will balance traffic across those remaining pods.  If one of the zones not hosting

**Comment by eparis** [comment]
@k8s-bot test this, issue #22435

**Comment by justinsb** [comment]
Closing in favor of https://github.com/kubernetes/kubernetes.github.io/pull/140

**Pull request closed without merging**